karma_config/vueLocal.js

```diff
@@ -0,0 +1,35 @@
+const vue = require('../node_modules/vue');
```

Comment:
The alternative would be to include the async intl stuff as well, put `vueLocal` at a higher level directory, then include it in both `karma.conf.js` and core?